service/collector/env.go

```diff
@@ -13,9 +18,17 @@ const (
 )
 
 var (
-	envDesc *prometheus.Desc = prometheus.NewDesc(
-		prometheus.BuildFQName(namespace, subsystemEnv, "inconsistencies"),
-		"Information of environment inconsistencies.",
+	envClusterWithoutFileDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystemEnv, "cluster_without_file"),
```

Comment:
Metric name should be `cluster_without_env_file` IMO.
I sometimes look through the available metrics in prometheus and `cluster_without_file` seems very ambiguous to me.

Reply:
(I know the description exists but in prometheus that description isn't shown when entering a query)

Reply:
Renamed.